grammar/gen-wat-parser.sh

```diff
@@ -1,15 +1,22 @@
 #! /usr/bin/env bash
 #
-ANTLR4=antlr-4.13.0-complete.jar
+ANTLR4=antlr-4.13.2-complete.jar
```

Comment:
Let's keep ANTLR 4.13.0, unless we want to change it globally with its runtime (specified in `build.sbt`).